### PR TITLE
Fixed typos and a few correction

### PR DESCRIPTION
**Pull request by chrishna1**

### CODE REVIEW

File: src/agents.jl [hunk 1]
```diff
@@ -67,9 +67,9 @@ of the *Environment*
 abstract type Percept end
 
 """
-Given a *Percept* returns an *Action* apt for the agent.
+Given a *Percept* ? returns an *Action* apt for the agent.
```
Comment (sambitdash) [inline src/agents.jl]: Not sure why the ? mark.

Comment (chrishna1) [reply src/agents.jl]: Sentence was not complete, and I wanted to put agent program there but wasn't sure then.

Comment (chrishna1) [reply src/agents.jl]: So I asked you. Sorry if it is not the proper way, will keep in mind later on.

File: src/search.jl [hunk 2]
```diff
@@ -544,3 +544,30 @@ function execute(search::GeneticAlgorithmSearch, problem)
...
     end
 end
+struct AND_ORGraphSearch{SQ,S<:State} <: SearchAlgorithm
+    SQ_t::Type
+    #AND_Search::SearchAlgorithm
+    AND_ORGraphSearch{SQ,S}() where {SQ,S<:State} = 
+
+end
+function OR_Search(::
+
```
Comment (sambitdash) [inline src/search.jl:554]: Looks like a WIP submission. Please complete the implementation and let me know when it's good to be published.

**Review by sambitdash — CHANGES_REQUESTED** [review]
This PR has Work-in-progress code. Unless completed cannot be approved.

**Pull request closed without merging**